Marker/Classes/Marker.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  Marker.swift
+//  Marker
+//
+//  Created by Htin Linn on 5/15/17.
+//  Copyright © 2017 Prolific Interactive. All rights reserved.
+//
+
+import Foundation
+
+/// Returns formatted Markdown text as an attributed string.
+///
+/// - Parameters:
+///   - markdownText: String with Markdown tags.
+///   - textStyle: Text style object containing style information.
+/// - Returns: Formatted Markdown text.
+public func attributedMarkdownString(from markdownText: String,
+                                     using textStyle: TextStyle) -> NSAttributedString {
+    guard let (parsedString, elements) = try? MarkdownParser.parse(markdownText) else {
+        return NSAttributedString(string: markdownText, textStyle: textStyle)
```

Comment:
Setting text is a very common operation that you do in your projects. I feel like it would be such as hassle to have to write `try` or `try?` every time you want to use a `Marker` function. What's being returned here is a better compromise to me: the text is still being set, along with the styles being applied to the entire string.